src/main/java/com/aws/greengrass/deployment/ThingGroupHelper.java

```diff
@@ -30,7 +32,8 @@ public class ThingGroupHelper {
     public static final String THING_GROUP_RESOURCE_TYPE_PREFIX = THING_GROUP_RESOURCE_TYPE + "/";
     private static final int DEFAULT_RETRY_COUNT = Integer.MAX_VALUE;
     static final List<Class> DEVICE_OFFLINE_INDICATIVE_EXCEPTIONS = Arrays.asList(SdkClientException.class,
```

Comment:
nit: you should rename this variable since it's no longer just offline exceptions